### ISSUE TITLE
Error when trying to patch US boot.elf

### ISSUE THREAD

**Issue by RadiantDaystar** [gibbed/DemonsSoulsDebug#5]
Getting this when running build.bat, changing the patch name doesn't help.

Unhandled Exception: System.IO.FileNotFoundException: Could not find file 'C:\Users\Infallible Daystar\Desktop\DSDB\debug.patch.elf'.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share)
   at System.IO.File.OpenRead(String path)
   at PatchElf.Program.Run(String inputPath, String patchPath, String outputPath, String verifyHash, Boolean addCave, Nullable`1 desiredCaveAddress, Nullable`1 desiredCaveSize, Nullable`1 originalCodeSize)
   at PatchElf.Program.Main(String[] args)

**Comment by gibbed** [comment]
Yeah I made a mistake when I pushed the new commits and hadn't had a chance to push a fix yet.

If you open `build.bat` in notepad/text editor of choice, change `debug.patch.elf` to `debug.elf.patch`, then save, it should resolve the issue.

**Comment by JacktheRipper-glitch** [comment]
I had a error like this as well, but my .bat is already set to that, so I am confused, I wasn't sure if this was because digital copies of the US version of demon souls were supported or not.

Unhandled Exception: System.IO.FileNotFoundException: Could not find file 'C:\Users\nsiuser\Desktop\demon souls debug stuff\debug.patch.elf'.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share)
   at System.IO.File.OpenRead(String path)
   at PatchElf.Program.Run(String inputPath, String patchPath, String outputPath, String verifyHash, Boolean addCave, Nullable`1 desiredCaveAddress, Nullable`1 desiredCaveSize, Nullable`1 originalCodeSize)
   at PatchElf.Program.Main(String[] args)

**Comment by gibbed** [comment]
If it's looking for `debug.patch.elf` you didn't modify the `.bat` file as suggested in my earlier comment. I still haven't pushed a fix for this yet.

**Comment by JacktheRipper-glitch** [comment]
> If it's looking for `debug.patch.elf` you didn't modify the `.bat` file as suggested in my earlier comment. I still haven't pushed a fix for this yet.

I am dumb, I am so sorry! I misread the .bat and thought it already had the right thing. Also are digital copies of the game supported or no?

**Comment by gibbed** [comment]
It's all good, were you able to sort it out?

Edit: digital copies should in theory be supported on PS3 directly. They run the same code (it's effectively an ISO image emulation... thing).

I don't know if RPCS3 ever got support for that though.

**Comment by JacktheRipper-glitch** [comment]
> It's all good, were you able to sort it out?
> 
> Edit: digital copies should in theory be supported on PS3 directly. They run the same code (it's effectively an ISO image emulation... thing).
> 
> I don't know if RPCS3 ever got support for that though.

Yeah I was able to patch it, I just can't get it to run, even turned it into a .self. I would also like to clarify I am trying to get it set up on an actual ps3, not on rpcs3. I did try doing a bunch of different things, including making the debug.elf into a debug.self, didn't work, tried the debug.elf replacing the eboot, tried making the debug.elf into an eboot, I could not get the game to start up with any of those methods.
